src/transformers/models/musicgen/convert_musicgen_transformers.py

```diff
@@ -212,6 +212,7 @@ def convert_musicgen_checkpoint(
         "`['facebook/musicgen-stereo-small', 'facebook/musicgen-stereo-medium', 'facebook/musicgen-stereo-large']` "
         "for the stereo checkpoints.",
     )
```

Comment:
IMO we just have to update the doc here no?

Reply:
To mention that it can also be a path to a custom checkpoint

Reply:
I think `decoder_config_from_checkpoint` will throw an error with a custom checkpoint

https://github.com/huggingface/transformers/blob/3cc28d141f895f071960a0d5dfce9554ba3739a3/src/transformers/models/musicgen/convert_musicgen_transformers.py#L105-L109

Is it possible to infer the config?

Reply:
We can update that error as well, the conversion script is not meant to only support official checkpoints.

Reply:
That makes sense. The script should be flexible enough to support all checkpoints. Can we expect the custom checkpoints to follow the same naming scheme? i.e. `musicgen-stereo-[small|medium|large]`